incidents.md

```diff
@@ -0,0 +1,110 @@
+        </ul>
+    <p>The following example shows what will be returned after refetching the transactions for the expected <code>originatorTransactionType</code>.</p>
+    <table style="text-align:left">
+                <thead>
+                    <tr>
+                      <th>Expected <code>originatorTransactionType</code></th>
+                      <th>Returned <code>originatorTransactionType</code></th>
+                    </tr>
+                </thead>
+                    <tbody>             
+                    <tr>
+                       <td>
+                            <pre>
+                             ...
+                                    {
+                                        "timestamp": "2021-03-30T16:03:31.003+0000",
+                                        "amount": -10,
+                                        "originatorTransactionType": "CARD_PAYMENT_FEE",
+                                        "originatingTransactionUuid": "ff4f492e-914c-1bbb-bb86-850e353b75b8"
+                                    },
+                                    {
+                                        "timestamp": "2021-03-30T16:03:31.000+0000",
+                                        "amount": 780,
+                                        "originatorTransactionType": "CARD_PAYMENT",
+                                        "originatingTransactionUuid": "ff4f492e-914c-1bbb-bb86-850e353b75b8"
+                                    },
+                               ...
+                            </pre>  
+                       </td>
+                       <td>
+                            <pre>
+                             ...
+                                    {
+                                        "timestamp": "2021-03-30T16:03:31.003+0000",
+                                        "amount": -10,
+                                        "originatorTransactionType": "PAYMENT_FEE",
+                                        "originatingTransactionUuid": "ff4f492e-914c-1bbb-bb86-850e353b75b8"
+                                    },
+                                    {
+                                        "timestamp": "2021-03-30T16:03:31.003+0000",
+                                        "amount": 780,
+                                        "originatorTransactionType": "PAYMENT",
+                                        "originatingTransactionUuid": "ff4f492e-914c-1bbb-bb86-850e353b75b8"
+                                    },
+                               ...
+                            </pre>  
+                       </td>
+                    </tr>   
+                    </tbody>
+            </table>  
+    </li>    
+    <li><p>If your integration disregards transactions with <code>originatorTransactionType</code> as <code>PAYMENT</code> and <code>PAYMENT_FEE</code>, handle those transactions and bookkeep the same as you used to do with <code>CARD</code>.</p></li> 
```

Comment:
How is it to handle those transactions? Any more details needed by the integrators?